paasta_tools/cli/schemas/adhoc_schema.json

```diff
@@ -61,6 +61,20 @@
                         "type": "string"
                     }
                 },
+                "crypto_keys": {
+                    "encrypt": {
+                        "type": "array",
+                        "items": {
+                            "type": "string"
```

Comment:
would it help to add a regex pattern to the schema for what the items should look like (e.g., the stuff in these arrays should match `[a-zA-Z]+`)?